studybuddies/src/app/app.module.ts

```diff
@@ -20,6 +19,9 @@ import { ViewClassTutorComponent } from './components/tutor/view-class-tutor/vie
 import { ErrorPageComponent } from './components/error-page/error-page.component';
 import { ConstructionPageComponent } from './components/construction-page/construction-page.component';
 import { LoginComponent } from './components/login/login.component';
+import { BaseURLInterceptorService } from './services/base-urlinterceptor.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202914)
```suggestion
import { HttpClientModule } from "@angular/common/http";
```